reana_server/config.py

```diff
@@ -38,6 +38,13 @@
 
 REANA_SSO_CERN_CONSUMER_SECRET = os.getenv("CERN_CONSUMER_SECRET", "CHANGE_ME")
 
+REANA_KUBERNETES_JOBS_MEMORY_LIMIT = os.getenv("REANA_KUBERNETES_JOBS_MEMORY_LIMIT", "4Gi")
```

Comment:
I've renamed it to `REANA_KUBERNETES_DEFAULT_JOBS_MEMORY_LIMIT` for now, since this is used only for the complexity estimation in r-server. That's why initially I left the value hardcoded in the code, since having the config could lead to some inconsistencies. But I guess changing the name as you suggested could be enough to avoid it

Reply:
I agree with @mvidalgarcia that it is better to set default in Helm values, should we want to use one. That would be the cleanest solution.

There may be some installations that don't want to use any RAM limit, so in theory this value could be "0 GiB"  by default for some.  We should be ready for this case when calculating priorities. (E.g. send all workflows to the highest bin for this case.)

So I'd suggest to reuse the Helm value here, and be ready for 0 GiB case (which is currently the default).

P.S. Alternatively, if this value is used only to calculate priorities, we may want to use whatever and rename the variable somewhat differently so that it wouldn't evoke the "hard limit" feeling. But I'd prefer to make it consistent with Helm value anyway...